### DISCUSSION TITLE
"Bookmark all tabs" to "Tabs collection" both temporary and non-exportable

### DISCUSSION THREAD

**Discussion by SandSublime222** [bromite/bromite#1140] [Q&A]
Tried to submit a bug report (https://github.com/bromite/bromite/issues/1138) but bot is flagging even though I am following the template (tried several edits, but does not open). Opening up a discussion here instead.

When using `Bookmark all tabs`, the bookmarks are saved to the `Tabs collection` root folder in bookmark manager (this was a recent change, previously they would go into `Mobile bookmarks`). 
- __Bug:__ These `Tabs collection` bookmarks are not included when exporting bookmarks from the bookmark manager.
- __Bug:__ These `Tabs collection` bookmarks disappear when users close and restart Bromite.
- __Note:__ bookmarks in `Mobile bookmarks` are preserved across sessions and are included in exported html file).

This is with:
Version: `90.0.4430.204`
Arch: `arm64`
No flags changed.

Is this the intended behavior?

**Comment by csagan5** [comment]
@SandSublime222 the discussions are not for bug reports; the template explains clearly what you have to do:

> If the bug is related to functionality that does not exist in Chromium then answer "No".
